pages/index.js

```diff
@@ -90,6 +90,15 @@ const Talks = () => {
 };
 
 const Communication = () => {
+    return (
```

Comment:
This component is useless now. You can directly add this in `<Home />` component
ie
```js
<h2>Communication</h2>
    <Grid>
         <NextTweet />
         <NoCommunication />
    </Grid>
```